source/whats-new.txt

```diff
@@ -57,6 +57,77 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.6:
+
+What's New in 6.6
+-----------------
+
+The {+driver-short+} v6.6 release includes the following features:
+
+- Adds the ``addStage()`` method to the fluid aggregation API. You can use this method to
+  add aggregation pipeline stages individually, as shown in the following
+  example:
+
+  .. code-block:: javascript
+
+     const documents = await users.aggregate().addStage({ $project: { name: true } }).toArray();
+
+- Adds the ``cause`` and ``dependencyName`` fields to the ``MongoMissingDependencyError``
+  class. You can use these fields to programmatically determine if a package is missing
+  or why a package didn't load.
+
+- Adds the ``minRoundTripTime`` property to the ``ServerDescription`` class. This
+  property contains the minimum round-trip time over the last 10 heartbeats.
+
+- Adds the ``toJSON()`` method to the ``TopologyDescription`` class. You can use this
+  method to stringify ``TopologyDescription`` objects to JSON.
+
+- Adds cursor support for the ``Collection.indexExists()``, ``Collection.indexes()``, and
+  ``Collection.indexInformation()`` methods in Typescript.
```

Comment:
This should be "Adds cursor **options** support", the options types previously did not reflect the underlying AbstractCursorOptions that are permitted on these APIs

- `IndexInformationOptions` https://github.com/mongodb/node-mongodb-native/blob/8be035b560c1ffa42e97394929069857a045b8a3/src/operations/indexes.ts#L75
- extends `ListIndexesOptions` https://github.com/mongodb/node-mongodb-native/blob/8be035b560c1ffa42e97394929069857a045b8a3/src/operations/indexes.ts#L315
- extends `AbstractCursorOptions` https://github.com/mongodb/node-mongodb-native/blob/8be035b560c1ffa42e97394929069857a045b8a3/src/cursor/abstract_cursor.ts#L70